scheduler/core/service/service.py

```diff
@@ -39,14 +42,16 @@ def __call__(self):
 
 
 def build_scheduler(start: Time = Time("2018-10-01 08:00:00", format='iso', scale='utc'),
-                    end: Time = Time("2018-10-03 08:00:00", format='iso', scale='utc')) -> Service:
+                    end: Time = Time("2018-10-03 08:00:00", format='iso', scale='utc'),
+                    sites: FrozenSet[Site] = ALL_SITES) -> Service:
```

Comment:
LOL I just picked these dates randomly but they look like they'll live on for awhile in infamy.